### PR TITLE
Replaces eye blur effect with real one

### PR DESCRIPTION
**Pull request by MoXuT0**
## About The Pull Request

Per title. Port of https://github.com/tgstation/tgstation/pull/41759

## Changelog
:cl:
add: Eye blur effect now uses byond's filters.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/status_procs.dm [hunk 1]
```diff
@@ -128,28 +128,24 @@
 
 /mob/living/proc/blur_eyes(amount)
 	if(amount>0)
-		var/old_eye_blurry = eye_blurry
 		eye_blurry = max(amount, eye_blurry)
-		if(!old_eye_blurry)
-			overlay_fullscreen("blurry", /obj/screen/fullscreen/blurry)
```
Comment (DominikPanic) [inline code/modules/mob/living/status_procs.dm]: If this overlay is not used elsewhere, you should remove it

Comment (MoXuT0) [reply code/modules/mob/living/status_procs.dm]: I've already removed it, it was used only here

**Comment by Rohesie** [comment]
What's the status on this PR?

File: code/modules/mob/living/carbon/life.dm [hunk 2]
```diff
@@ -273,7 +273,8 @@
 		adjust_blindness(-1)
 	if(eye_blurry)			//blurry eyes heal slowly
 		adjust_blurriness(-1)
-
+		if(client)
```
Comment (DominikPanic) [inline code/modules/mob/living/carbon/life.dm]: This check is redundant since update_eye_blur already checks for client

**Comment by MoXuT0** [comment]
@Rohesie I ded

**Pull request merged**